### ISSUE TITLE
drivers/generic/generic.go defaultTimeout is too small

### ISSUE THREAD

**Issue by vigasin** [docker/machine#4008]
Here's what I see:

$ for ((i=0;i<100;i++));
do
   echo -ne "$i "
   docker-machine env cloud-dev >& $i.txt || echo "Failed"
   echo
done
......
35 Failed
......
77 Failed
......

i.e. on every run I always see some fails.

I'm located in LA and the machine cloud-dev is in the AWS us-east-1 region.

Sometimes 1 second is just not enough to set up a TCP connection.

I guess it's worth to increase it.

**Comment by nathanleclaire** [comment]
Hm, I thought it was higher.

If it is only 1 second that does seem a bit aggressive to me -- would be a good opportunity to contribute a PR, maybe something more like 10 seconds :)

**Comment by vigasin** [comment]
Yep, created a pull request for this: https://github.com/docker/machine/pull/4030
